ssa.go

```diff
@@ -195,7 +201,7 @@ func ReadFromSSA(i io.Reader) (o *Subtitles, err error) {
 		// Split on ":"
 		var split = strings.Split(line, ":")
 		if len(split) < 2 || split[0] == "" {
-			log.Printf("astisub: not understood: '%s', ignoring", line)
+			opts.OnInvalidLine(line)
```

Comment:
Once `OnInvalidLine` is a field, you'll need to wrap this with `if opts.OnInvalidLine != nil`